de.metas.document.archive/de.metas.document.archive.base/src/main/sql/postgresql/system/5451430_sys_724_AddIsInvoiceEmailEnabled.sql

```diff
@@ -0,0 +1,69 @@
+INSERT INTO AD_Element (AD_Client_ID,AD_Element_ID,AD_Org_ID,ColumnName,Created,CreatedBy,EntityType,IsActive,Name,PrintName,Updated,UpdatedBy) VALUES (0,543189,0,'IsInvoiceEmailEnabled',TO_TIMESTAMP('2016-09-29 10:26:10','YYYY-MM-DD HH24:MI:SS'),100,'de.metas.document.archive','Y','Invoice Email Enabled','Invoice Email Enabled',TO_TIMESTAMP('2016-09-29 10:26:10','YYYY-MM-DD HH24:MI:SS'),100)
+;
+
+-- 29.09.2016 10:26
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Element_Trl (AD_Language,AD_Element_ID, Description,Help,Name,PO_Description,PO_Help,PO_Name,PO_PrintName,PrintName, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language,t.AD_Element_ID, t.Description,t.Help,t.Name,t.PO_Description,t.PO_Help,t.PO_Name,t.PO_PrintName,t.PrintName, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Element t WHERE l.IsActive='Y' AND l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N' AND t.AD_Element_ID=543189 AND NOT EXISTS (SELECT * FROM AD_Element_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Element_ID=t.AD_Element_ID)
+;
+
+-- 29.09.2016 10:26
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Column (AD_Client_ID,AD_Column_ID,AD_Element_ID,AD_Org_ID,AD_Reference_ID,AD_Table_ID,AllowZoomTo,ColumnName,Created,CreatedBy,DDL_NoForeignKey,DefaultValue,EntityType,FieldLength,IsActive,IsAdvancedText,IsAllowLogging,IsAlwaysUpdateable,IsAutocomplete,IsCalculated,IsDimension,IsEncrypted,IsGenericZoomKeyColumn,IsGenericZoomOrigin,IsIdentifier,IsKey,IsLazyLoading,IsMandatory,IsParent,IsSelectionColumn,IsStaleable,IsSyncDatabase,IsTranslated,IsUpdateable,IsUseDocSequence,Name,SeqNo,Updated,UpdatedBy,Version) VALUES (0,555107,543189,0,20,540453,'N','IsInvoiceEmailEnabled',TO_TIMESTAMP('2016-09-29 10:26:40','YYYY-MM-DD HH24:MI:SS'),100,'N','Y','de.metas.document.archive',1,'Y','N','Y','N','N','N','N','N','N','N','N','N','N','Y','N','N','N','N','N','Y','N','Invoice Email Enabled',0,TO_TIMESTAMP('2016-09-29 10:26:40','YYYY-MM-DD HH24:MI:SS'),100,0)
+;
+
+-- 29.09.2016 10:26
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+INSERT INTO AD_Column_Trl (AD_Language,AD_Column_ID, Name, IsTranslated,AD_Client_ID,AD_Org_ID,Created,Createdby,Updated,UpdatedBy) SELECT l.AD_Language,t.AD_Column_ID, t.Name, 'N',t.AD_Client_ID,t.AD_Org_ID,t.Created,t.Createdby,t.Updated,t.UpdatedBy FROM AD_Language l, AD_Column t WHERE l.IsActive='Y' AND l.IsSystemLanguage='Y' AND l.IsBaseLanguage='N' AND t.AD_Column_ID=555107 AND NOT EXISTS (SELECT * FROM AD_Column_Trl tt WHERE tt.AD_Language=l.AD_Language AND tt.AD_Column_ID=t.AD_Column_ID)
+;
+
+-- 29.09.2016 10:26
+-- I forgot to set the DICTIONARY_ID_COMMENTS System Configurator
+ALTER TABLE C_Doc_Outbound_Log ADD IsInvoiceEmailEnabled CHAR(1) DEFAULT 'Y' CHECK (IsInvoiceEmailEnabled IN ('Y','N')) NOT NULL
```

Comment:
please move all the DDL to the bottom, and add a commit between DML and DDL.
otherwise there might be an error on migration (sth with "pending" trigger events")